user-docs/documentation/docs/getting-started/installation.md

```diff
@@ -26,23 +26,22 @@ If you want to take is as an example to start your own project,
 
 - Go to the shared configuration file in `packages/serverless-configuration/src/sharedConfig.ts`
 - Change the `projectName` variable to the name of your project (note that this name must not be too long)
-- Change the dev profile name to one that suits you
-- Search and replace "swarmion-starter" and replace it with the same project name
```

Comment:
We need to keep this step

Reply:
I cannot find occurrences of `swarmion-starter`, do you mean `swarmion`?

Reply:
No it is `swarmion-starter`, it is meant to apply to https://github.com/swarmion/template